### PR TITLE
Open variation preview in a new tab

### LINKED ISSUE #40173: [Fix] Open the variation preview in a new tab
When the user selects `Preview` from the kebab menu in the variations list, we open the product page in the same tab. Meanwhile, it should open in a new tab to avoid interrupting the user.

image

**Acceptance criteria**

- [ ] Clicking Preview in the quick actions menu opens the preview in a new tab.

### PR DESCRIPTION
**Pull request by octaedro**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR adds the code to open the variation preview in a new tab.

**Acceptance criteria**

- [ ] Clicking Preview in the quick actions menu opens the preview in a new tab.

Closes #40173.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that the `New product editor` feature is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. Ensure that the `product-variation-management` feature is enabled under the `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper`.
3. Navigate to `/wp-admin/admin.php?page=wc-admin&path=/add-product&tab=variations` and click the `Add variation options` button from the `Variations` tab.
4. After creating the variations, click the three dots icon in the line of the first variation and select `Preview`.
5. This action should open a new tab with the selected variation.

Screen Capture on 2023-09-25 at 16-16-21

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ export function VariationActionsMenu( {
 					>
 						<MenuItem
 							href={ selection.permalink }
+							target="_blank"
```
Comment (mdperez86) [inline packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx:48]: When adding `target="_blank"` we should also add `rel="noreferrer"` for security reasons.

**Review by mdperez86 — CHANGES_REQUESTED** [review]
I left one small comment @octaedro.

File: packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx (hunk 1 above)
Comment (octaedro) [reply packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx:48]: Good call @mdperez86! I added that in the commit [2723566](https://github.com/woocommerce/woocommerce/pull/40414/commits/2723566ecb6b6a74749a98dce706b930cdad34ff).
Could you take another look at this PR?

**Review by mdperez86 — APPROVED** [review]
LGTM @octaedro nice work!!

**Pull request merged**